lib/models/messages/twitch/event.dart

```diff
@@ -23,3 +23,19 @@ class TwitchFollowEventModel extends MessageModel {
     required bool pinned,
   }) : super(messageId: messageId, pinned: pinned);
 }
+
+class TwitchCheerEventModel extends MessageModel {
+  final int bits;
+  final bool isAnonymous;
+  final String cheerMessage;
+  final String giverName;
```

Comment:
If the cheer is anonymous, is `giverName` null? Seems like it might be easier to make `giverName` nullable and make `isAnonymous` a getter?

Reply:
both `isAnonymous` and `giverName` are always supplied in Firestore. The name is present in the database even if `isAnonymous` is `true`

Reply:
Wow that's surprising that Twitch would expose that. Haha okay :)